.changeset/few-planets-brush.md

```diff
@@ -0,0 +1,5 @@
+---
+'slate-react': minor
```

Comment:
This part is counter-intuitive, but since we're pre-`1.0` we use `patch` for non-breaking releases for now.